website/tantalus/templates/admin/borrel/borrelreservation/submit_line.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'admin/submit_line.html' %}
+{% load i18n admin_urls %}
+<div class="submit-row">
+{% block submit-row %}
+{% if show_save %}<input type="submit" value="Save" class="default" name="_save">{% endif %}
+{% if show_delete_link and original %}
+    {% url opts|admin_urlname:'delete' original.pk|admin_urlquote as delete_url %}
+    <p class="deletelink-box"><a href="{% add_preserved_filters delete_url %}" class="deletelink">Delete</a></p>
+{% endif %}
+{% if show_push_to_tantalus %}
+    <input type="submit" value="Push to Tantalus" name="_pushtotantalus" id="push-to-tantalus">
+{% endif %}
+{% if show_force_push_to_tantalus %}
+    <input type="submit" value="Force push to Tantalus" name="_pushtotantalus" id="force-push-to-tantalus">
+    <script>
+        const force_push_button = document.getElementById('force-push-to-tantalus');
```

Comment:
Is a force push option really needed or will it only create problems when force pushing? I mean if there is a problem the treasurer can simply add the order manually right?

Reply:
Good point! I'll ask Douwe in the break!

Reply:
I will rewrite to https://github.com/JobDoesburg/django-easy-admin-object-actions soon ™   and then we will remove large parts of this code again anyhow